css-three-column-layout/styles.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  color: white;
+  font-family: sans-serif, 'Courier New', Courier;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  background-color: rgb(25, 150, 218);
+}
+
+.row {
+  display: flex;
+  padding: 0 70px;
+}
+
+.column-full {
+  display: flex;
+  padding-top: 60px;
+}
+
+.column-full h1 {
+  font-weight: lighter;
+  margin-bottom: 25px;
+}
+
+.column-third {
+  width: 33%;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 50px;
```

Comment:
Make this a separate class, only `width` or `flex-basis` should be applied to `column` classes.